homeassistant/components/tolosauna/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "domain": "tolosauna",
+  "name": "TOLO Sauna",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/tolosauna",
+  "requirements": [
+    "tololib==0.1.0a6"
+  ],
+  "dependencies": [],
```

Comment:
```suggestion
```

Reply:
Just kept it as it was created by HA automatically. Removed it now.